torch/csrc/jit/python/pybind_utils.h

```diff
@@ -46,12 +46,21 @@
 namespace torch {
 namespace jit {
 
+py::object toPyObject(IValue ivalue);
+
 // The PythonFutureWrapper for ivalue::Future
-struct PythonFutureWrapper {
-  explicit PythonFutureWrapper(c10::intrusive_ptr<c10::ivalue::Future> fut)
-      : fut(std::move(fut)) {}
+struct VISIBILITY_HIDDEN PythonFutureWrapper {
```

Comment:
Let's add a comment here to explain the reason that we added the `VISIBILITY_HIDDEN `

Reply:
Adding.